kubecontrolplane/install.go

```diff
@@ -0,0 +1,26 @@
+package kubecontrolplane
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	kubecontrolplanev1 "github.com/openshift/api/kubecontrolplane/v1"
+)
+
+const (
+	GroupName = "kubecontrolplane.config.openshift.io"
+)
+
+var (
+	schemeBuilder = runtime.NewSchemeBuilder(kubecontrolplanev1.Install)
+	// Install is a function which adds every version of this group to a scheme
+	Install = schemeBuilder.AddToScheme
+)
+
+func Resource(resource string) schema.GroupResource {
```

Comment:
Do we need Resource? I think I saw that dropped in kube types for some time.

Reply:
either Kind() or Resource() was removed, I remember that when we were switching stuff to use external api

Reply:
maybe specific to external apis?